youtube_dl/extractor/vimeo.py

```diff
@@ -730,12 +730,12 @@ def _title_and_entries(self, list_id, base_url):
             # Try extracting href first since not all videos are available via
             # short https://vimeo.com/id URL (e.g. https://vimeo.com/channels/tributes/6213729)
             clips = re.findall(
-                r'id="clip_(\d+)"[^>]*>\s*<a[^>]+href="(/(?:[^/]+/)*\1)', webpage)
+                r'id="clip_(\d+)"[^>]*>\s*<a[^>]+href="(/(?:[^/]+/)*\1)[^>]*title="([^"]+)"', webpage)
```

Comment:
Will break the extraction if there is no title attribute. This should be optional.